### DISCUSSION TITLE
How to enable n2n Streaming via Unity SDK?

### DISCUSSION THREAD

**Discussion by tyler-lan-party** [ant-media/Ant-Media-Server#5701] [FAQs]
Regarding the Unity SDK, how do we enable 'multicast' (n2n) mode for streaming?

This 'n2n' functionality is mentioned in the overview:
https://antmedia.io/webrtc-signaling-servers-everything-you-need-to-know/

However, after implementing the demo, and with an active AMS Enterprise instance running on Linode, I can't get +3 users to share video streams.

As my working reference example, this latest blog post demonstrates "peer 2 peer" mode via the final dropdown selection:
https://antmedia.io/how-to-use-the-new-ant-media-server-unity-sdk/

Notably, a similar (earlier) blog post shows this final dropdown selection as 'Multi_sendrecv' and implies a 'conference call':
https://antmediaserver.medium.com/webrtc-streaming-in-unity-easy-guide-to-multiplatform-streaming-be3b45e081cb

How should I proceed?

Thank you in advance!

**Comment by tremzo** [comment]
@tyler-lan-party Can you provide some more details about what you mean by "I can't get +3 users to share video streams"? Are there any errors you are seeing? Can you tell me exactly what you've implemented and what you want to achieve?

**Comment by lastpeony** [comment]
Conference on unity is not supported. Multi_sendrecv medium post is deprecated
